Marlin/Configuration_adv.h

```diff
@@ -2071,6 +2071,30 @@
   //#define FILAMENT_LCD_DISPLAY
 #endif
 
+/**
+ * Power Monitor
+ * Monitor the voltaqe and/or current drawn and/or power used
+ *
+ * If enabled then configurable and readable using GCodes M430, M431 and M432 (current, voltage and power resp)
+ *
+ * The current sensor feeds DC voltage (relative to the measured current) to an analog pin
+ * The voltage sensor feeds DC voltage (relative to the measured voltage) to an analog pin
+ */
+//#define POWER_MONITOR_CURRENT   // Monitor the system current
+//#define POWER_MONITOR_VOLTAGE   // Monitor the system voltage
+#if EITHER(POWER_MONITOR_CURRENT, POWER_MONITOR_VOLTAGE)
+  #define POWER_MONITOR_VOLTS_PER_AMP   0.05000   // Input voltage to the MCU analog pin per amp  - DO NOT exceed the ADC reference voltage!
+  #define POWER_MONITOR_VOLTS_PER_VOLT  0.11786   // Input voltage to the MCU analog pin per volt - DO NOT exceed the ADC reference voltage!
+
+  #define POWER_MONITOR_FIXED_VOLTAGE   13.6      // used as the voltage if you use a current sensor with no voltage sensor and want power display
+
+  // ADC reference voltage appropriate to the board. Suggested values:
+  //   3.0 : Possible ADC reference voltage on 32-bit MCUs
+  //   3.3 : Typical ADC reference voltage on 32-bit MCUs
+  //   5.0 : Typical ADC reference voltage on 8-bit MCUs
+  #define POWER_MONITOR_ADC_VREF 3.3
```

Comment:
This should match the `MOTHERBOARD` setting in the corresponding `Configuration.h` file. Since the `MOTHERBOARD` is a generic RAMPS 1.4, the default should be 5.0.

Reply:
oops sorry, I didn't mean to change that to 3.3 default.

How do you start these conversation in github ?    I haven't yet worked that out